instructionAPI/src/Expression.C

```diff
@@ -78,6 +78,19 @@ namespace Dyninst
                     assert(!"unexpected machine register size!");
             }
         }
+        Expression::Expression(std::vector<MachRegister> rs) :
+            Expression::Expression(rs.size() * rs[0].size())
```

Comment:
sum the size of the elements and use that value as that makes this more generic. If the registers are the same type (and size) and then the constructor `Expression::Expression(MachRegister r, uint32_t len)` is used, right?  In the body you'd have to create a result and use setValue:

```cpp
Result result{totalSize};
this->setValue(result);
```